src/plugins/controls/public/options_list/components/options_list_popover_invalid_selections.tsx

```diff
@@ -40,18 +75,20 @@ export const OptionsListPopoverInvalidSelections = () => {
           )}
         </label>
       </EuiTitle>
-      {invalidSelections?.map((ignoredSelection, index) => (
-        <EuiFilterSelectItem
-          data-test-subj={`optionsList-control-ignored-selection-${ignoredSelection}`}
-          checked={'on'}
-          className="optionsList__selectionInvalid"
-          key={index}
-          onClick={() => dispatch(deselectOption(ignoredSelection))}
-          aria-label={OptionsListStrings.popover.getInvalidSelectionAriaLabel(ignoredSelection)}
-        >
-          {`${ignoredSelection}`}
-        </EuiFilterSelectItem>
-      ))}
+      <EuiSelectable
+        aria-label={OptionsListStrings.popover.getInvalidSelectionsSectionAriaLabel(
+          fieldName,
+          invalidSelections?.length ?? 0
+        )}
+        options={selectableOptions}
+        listProps={{ onFocusBadge: false, isVirtualized: false }}
```

Comment:
Because `isVirtualized` is false for the invalid selections, it will prevent double scrollbars in situations where both `optionCount > 7` and `invalidSelectionCount > 7`.

![Jan-10-2023 09-25-47](https://user-images.githubusercontent.com/8698078/211686700-d3f1c4d3-d18d-47f4-97fd-47aac6ac95aa.gif)